design/networking-blog-post.md

```diff
@@ -0,0 +1,73 @@
+# A New Network Stack for Zcash
+
+In our [2020 Engineering Roadmap][roadmap], we gave an overview of our plans
+for Zebra.  Announced last summer at Zcon1, Zebra aims to support the core
+strength of Zcash – its best-in-class cryptography – by placing it on a solid
+foundation, providing a modern, modular implementation that can be broken into
+components and used in many different contexts.  In that post, we briefly
+described the new network stack we designed and implemented for Zebra.  As a
+fork of Bitcoin, Zcash inherited Bitcoin's network protocol; in this post,
+we'll do a deep dive on Zebra's network stack.
+
+The new design is based on service-oriented architecture concepts from the
+[tower library][tower], used in Buoyant's [linkerd][linkerd].  It isolates the
```

Comment:
BTW: if you want to reference more production users of tower, either here or later in the post, I can give you some more names! :)